spec/factories.rb

```diff
@@ -29,18 +33,33 @@
   end
 
   factory :move do
-    sequence(:name) { |n| "Sybil #{n}th" }
+    sequence(:name) { |n| "#{Faker::App.name} #{rand(1000)}th" }
     user
     description "My favorite move"
   end
 
   factory :video do
-    sequence(:name) { |n| "Classic #{n}th" }
+    sequence(:name) { |n| "#{Faker::App.name} #{rand(1000)}" }
```

Comment:
[Some goes for this one](https://github.com/davidpdrsn/CardistryIO/pull/154/files#r69469507)